wdl-engine/src/inputs.rs

```diff
@@ -616,18 +646,50 @@ impl Inputs {
         let reader = BufReader::new(file);
         let object = mem::take(
             serde_json::from_reader::<_, JsonValue>(reader)
-                .with_context(|| {
-                    format!("failed to parse input file `{path}`", path = path.display())
-                })?
+                .context("failed to parse input file")?
                 .as_object_mut()
-                .with_context(|| {
-                    format!(
-                        "expected input file `{path}` to contain a JSON object",
-                        path = path.display()
-                    )
-                })?,
+                .context("expected input file to contain a JSON object")?,
         );
 
+        Self::parse_object(document, object).context("failed to parse input file")
```

Comment:
Similar to the above comments, I think we should include the file path in this error.

```suggestion
        Self::parse_object(document, object).with_context(|| format!("failed to parse input file `{path}`", path = path.display()))
```